lib/davinci_dtr_test_kit/profiles/nutrition_order/nutrition_order_validation.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../../validation_test'
+
+module DaVinciDTRTestKit
+  class NutritionOrderValidationTest < Inferno::Test
+    include DaVinciDTRTestKit::ValidationTest
+
+    title 'NutritionOrder resources returned during previous tests conform to the CRD NutritionOrder'
+    description %(
+This test verifies resources returned from the first read conform to
+the [CRD NutritionOrder](https://hl7.org/fhir/us/davinci-crd/STU2/StructureDefinition-profile-nutritionorder).
+Systems must demonstrate at least one valid example in order to pass this test.
+
+It verifies the presence of mandatory elements and that elements with
+required bindings contain appropriate values. CodeableConcept element
+bindings will fail if none of their codings have a code/system belonging
+to the bound ValueSet. Quantity, Coding, and code element bindings will
+fail if their code/system are not found in the valueset.
+
+    )
+
+    id :nutrition_order_validation
+    input :nutrition_order_resources
+
+    def resource_type
+      'NutritionOrder'
+    end
+
+    run do
+      perform_request_validation_test(nutrition_order_resources, resource_type,
```

Comment:
looks like this method adds errors to the message set, but doesn't cause the inferno test to fail, so we need to do that step after it. Will be similar to [this code](https://github.com/inferno-framework/davinci-dtr-test-kit/blob/355816d3c02ed74baa7356e9edce628645740c3d/lib/davinci_dtr_test_kit/payer_server_groups/payer_server_next_request_validation_test.rb#L55), but should be a failure, not a skip.

Reply:
I'd like to create a separate method for this and then create a unit test for it.  The idea is that you can create an [anonymous class](https://github.com/inferno-framework/us-core-test-kit/blob/main/spec/us_core/search_test_spec.rb#L24) with a run method and any associated details, and then [run it](https://github.com/inferno-framework/us-core-test-kit/blob/main/spec/us_core/search_test_spec.rb#L107). See [this Subscriptions PR](https://github.com/inferno-framework/subscriptions-test-kit/pull/12) for some other infrastructure I'm trying to setup around unit testing and get feedback on that may be helpful. Note that you'll need to stub out calls to the validator [as done here](https://github.com/inferno-framework/davinci-dtr-test-kit/blob/main/spec/davinci_dtr_test_kit/dtr_payer_server_questionnaire_package_group_static_response_validation_spec.rb#L66).